src/test/java/com/commercetools/sync/taxcategories/TaxCategorySyncOptionsBuilderTest.java

```diff
@@ -0,0 +1,137 @@
+package com.commercetools.sync.taxcategories;
```

Comment:
I addressed some small variable naming inconsistencies on this file, please consider changing the `TaxCategorySyncOptionsTest` same issues already exist in there.

Reply:
Hi
Typical copy-paste mistake, thank you for pointing it out.
Fixed: https://github.com/commercetools/commercetools-sync-java/pull/415/commits/ab3ec07ea8d19922beac5306ca179f1b3bfc2a9c